.github/workflows/python-app.yml

```diff
@@ -37,6 +37,9 @@ jobs:
           git merge "release-${{ env.RELEASE_VERSION }}" --no-edit
           git push
 
```

Comment:
why the space?

Reply:
I wanted to differentiate the two for readability. We can remove the space if it seems unnecessary